.github/workflows/acceptance.yml

```diff
@@ -14,6 +14,19 @@ on:
   pull_request:
 
 jobs:
+  files-changed:
+    name: detect what files changed
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check for backend file changes
+        uses: getsentry/paths-filter@v2
+        id: changes
+        with:
+          token: ${{ github.token }}
+          filters: .github/file-filters.yml
```

Comment:
Now, only one job will make these API calls, thus, hopefully reducing the chances of `ERROR: Bad credentials`:
![image](https://user-images.githubusercontent.com/44410/162493296-3cc0920c-347f-4eb8-91fc-140a5dbb6c8f.png)